CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- No changes yet.
+### Added
+- tchannel: add TLS support for the inbound. Supports accepting
+  TLS and plaintext connections on the same port.
```

Comment:
may this is less verbose? `tchannel: add TLS and plaintext inbound support on the same listening port`

Reply:
Aiming to keep similar change log as published earlier - https://github.com/yarpc/yarpc-go/blob/dev/CHANGELOG.md#added